common/vcenter_manage_domain_user.yml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Add or delete a domain user on vCenter Server
```

Comment:
Maybe indicate which input parameter is required here. 
Or move line 27 task  to the top. Then we can clearly konw which input parameter is required.

Reply:
Added (optional) to optional parameters